src/modelmanager.cpp

```diff
@@ -205,9 +205,24 @@ Status ModelManager::startFromConfig() {
         return StatusCode::UNKNOWN_ERROR;
     }
 
+    Status status = StatusCode::OK;
+
+    // Reading metric config only once per server start
+    if (!this->metricConfigLoadedOnce) {
```

Comment:
We are sure that this happens only once. This is from CLI without config.json

Reply:
Fixed.

Reply:
It won;t work if we start with passing in CLI --model_name --model_path?

This logic is invoked only on --config_path I think.

Reply:
Yes fixed.